product_pricelist_direct_print/wizards/product_pricelist_print_view.xml

```diff
@@ -94,6 +94,11 @@
                     <page string="Summary">
                         <field name="summary" nolabel="1" />
                     </page>
+                    <page string="Export Options">
+                        <group>
+                            <field name="breakage_per_category" />
```

Comment:
you define in orm show_product_images, but add breakage_per_category in the view.

are you sure ?

Reply:
Might have gone wrong when fixing merge conflicts.  Will look into it.